src/user/UserCredentials.tsx

```diff
@@ -121,9 +124,12 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
     useState<CredentialRepresentation>({});
   const [isResetPassword, setIsResetPassword] = useState(false);
   const [showData, setShowData] = useState(false);
-  const [isUserLabelEdit, setIsUserLabelEdit] = useState(false);
   const [editedUserCredential, setEditedUserCredential] =
     useState<CredentialRepresentation>({});
+  const [isUserLabelEdit, setIsUserLabelEdit] = useState({
+    status: false,
+    rowKey: "",
+  });
```

Comment:
This means though that you can only open one at a time.
```suggestion
  const [isUserLabelEdit, setIsUserLabelEdit] = useState<{status: boolean; rowKey: string}>({});
```

Reply:
Thanks @edewit I'll try to make them so that more than one can be open